### PR TITLE
Update RESOURCES.md

### PR DESCRIPTION
**Pull request by thiago-glauco**
https://metacpan.org/pod/distribution/Perl-Tidy/docs/stylekey.pod is no more reachable. Changed reference to https://metacpan.org/dist/Perl-Tidy/view/bin/perltidy

### CODE REVIEW

**Comment by thiago-glauco** [comment]
corrected according configlet recomendations.

**Comment by thiago-glauco** [comment]
Please, run again.

**Comment by SleeplessByte** [comment]
@exercism/perl5 anyone around to approve the workflow?

**Comment by kotp** [comment]
The commit itself is not only about what the title of this PR is about.  Can you restrict it to the topic of the PR?

**Review by kotp — CHANGES_REQUESTED** [review]
Thanks for the PR.  Some suggested changes, from spelling to content improvement (hopefully).

File: config.json [hunk 1]
```diff
@@ -654,7 +654,38 @@
...
+    },
+    {
+      "icon": "powerful",
+      "title": "30 years of Development",
+      "content": "Perl 5 is powerful and stable, with more than 30 years of development and increasing features."
+    },
+    {
+      "icon": "general-purpose",
+      "title": "General purpose",
+      "content": "Perl 5 can be used for a wide variety of workloads, from system monitoring to backend services."
```
[the remark this answers is not in the document]
Comment (thiago-glauco) [reply config.json:676]:
```suggestion
      "content": "Perl 5 can be used for a wide variety of workloads, from backend services to data analysis."
```

[the remark this answers is not in the document]
Comment (thiago-glauco) [reply config.json:676]:
```suggestion
      "content": "Perl 5 can be used for a wide variety of workloads, from backend services to speech translation."
```

[the remark this answers is not in the document]
Comment (thiago-glauco) [reply config.json:676]: I got your point, kotp. People must see a wide range of possibilities using some language. Perl is much like Python or TCL, supporting almost any workload.

**Comment by glennj** [comment]
I'm able to reproduce what's happening by rollowing the `- run:` sequence in the generator.yml workflow:

```sh
$ cd path/to/track/perl5
$ gh repo clone exercism/problem-specifications ./.problem-specifications -- --depth=1
$ carton install
$ cpm install App::Yath
$ carton exec -- yath start
$ carton exec -- yath run t/generator

Found: /home/glennj/src/exercism/tracks/perl5/.glennj-exercism-vm-yath-persist.json
  PID: 81695
  Dir: /tmp/yath-81694-VEKnkq
( STDERR )  job  1    Could not load Eval::TypeTiny at /home/glennj/src/exercism/tracks/perl5/local/lib/perl5/MooseX/MungeHas.pm line 24.                                                                                                     
( STDERR )  job  1    BEGIN failed--compilation aborted at /home/glennj/src/exercism/tracks/perl5/local/lib/perl5/MooseX/MungeHas.pm line 24.                                                                                                 
( STDERR )  job  1    Compilation failed in require at (eval 296) line 1.
( STDERR )  job  1    BEGIN failed--compilation aborted at /home/glennj/src/exercism/tracks/perl5/local/lib/perl5/Template/Mustache.pm line 8.                                                                                                
( STDERR )  job  1    Compilation failed in require at (eval 199) line 1.
( STDERR )  job  1    BEGIN failed--compilation aborted at /home/glennj/src/exercism/tracks/perl5/lib/Exercism/Generator.pm line 10.                                                                                                          
( STDERR )  job  1    Compilation failed in require at (eval 137) line 1.
( STDERR )  job  1    BEGIN failed--compilation aborted at t/generator/methods.t line 3.
( FAILED )  job  1    t/generator/methods.t
< REASON >  job  1    Test script returned error (Err: 2)
< REASON >  job  1    No plan was declared, and no assertions were made.
( STDERR )  job  2    Could not load Eval::TypeTiny at /home/glennj/src/exercism/tracks/perl5/local/lib/perl5/MooseX/MungeHas.pm line 24.                                                                                                     
( STDERR )  job  2    BEGIN failed--compilation aborted at /home/glennj/src/exercism/tracks/perl5/local/lib/perl5/MooseX/MungeHas.pm line 24.                                                                                                 
( STDERR )  job  2    Compilation failed in require at (eval 296) line 1.
( STDERR )  job  2    BEGIN failed--compilation aborted at /home/glennj/src/exercism/tracks/perl5/local/lib/perl5/Template/Mustache.pm line 8.                                                                                                
( STDERR )  job  2    Compilation failed in require at (eval 199) line 1.
( STDERR )  job  2    BEGIN failed--compilation aborted at /home/glennj/src/exercism/tracks/perl5/lib/Exercism/Generator.pm line 10.                                                                                                          
( STDERR )  job  2    Compilation failed in require at (eval 137) line 1.
( STDERR )  job  2    BEGIN failed--compilation aborted at t/generator/renders.t line 3.
( FAILED )  job  2    t/generator/renders.t
< REASON >  job  2    Test script returned error (Err: 2)
< REASON >  job  2    No plan was declared, and no assertions were made.

The following jobs failed:
+--------------------------------------+-----------------------+
| Job ID                               | Test File             |
+--------------------------------------+-----------------------+
| 9BD45F02-3299-11ED-9868-4C023139F9D0 | t/generator/renders.t |
| 9BD450D4-3299-11ED-9868-4C023139F9D0 | t/generator/methods.t |
+--------------------------------------+-----------------------+

                                Yath Result Summary
-----------------------------------------------------------------------------------
     Fail Count: 2
     File Count: 2
Assertion Count: 0
      Wall Time: 0.79 seconds
       CPU Time: 0.97 seconds (usr: 0.33s | sys: 0.06s | cusr: 0.51s | csys: 0.07s)
      CPU Usage: 123%
    -->  Result: FAILED  <--
```

Seeing the missing `Eval::TypeTiny`:
```sh
$ cpm install Eval::TypeTiny
DONE install Exporter-Tiny-1.004001
DONE install Type-Tiny-1.016010
2 distributions installed.
$ carton exec -- yath run t/generator

Found: /home/glennj/src/exercism/tracks/perl5/.glennj-exercism-vm-yath-persist.json
  PID: 81695
  Dir: /tmp/yath-81694-VEKnkq
( PASSED )  job  1    t/generator/methods.t
( PASSED )  job  2    t/generator/renders.t

                                Yath Result Summary
-----------------------------------------------------------------------------------
     File Count: 2
Assertion Count: 10
      Wall Time: 1.93 seconds
       CPU Time: 2.10 seconds (usr: 0.40s | sys: 0.03s | cusr: 1.38s | csys: 0.29s)
      CPU Usage: 108%
    -->  Result: PASSED  <--
```

## Reccommendation:

Either
1. add `requires 'Eval::TypeTiny';` to `cpanfile`, or
2. add `- run: cpm install Eval::TypeTiny` to `.github/workflows/generator.yml`

**Comment by glennj** [comment]
However, that doesn't fix the problem: recent changes to problem-specifications break the generation.
```sh
$ carton exec -- yath run t/files 2>&1 | tee generator.output
$ cat generator.output
Found: /home/glennj/src/exercism/tracks/perl5/.glennj-exercism-vm-yath-persist.json
  PID: 81695
  Dir: /tmp/yath-81694-VEKnkq
[  FAIL  ]  job  1  + accumulate: test suite matches generated
[  DEBUG ]  job  1    t/files/generated-tests.t line 20
(  DIAG  )  job  1    +------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    | PATH | GOT                          | OP | CHECK                        |
(  DIAG  )  job  1    +------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    | [8]  | use Accumulate qw<accumulate | eq | use Accumulate   qw<accumula |
(  DIAG  )  job  1    |      | >;                           |    | te>;                         |
(  DIAG  )  job  1    +------+------------------------------+----+------------------------------+
[  FAIL  ]  job  1  + clock: test suite matches generated
[  DEBUG ]  job  1    t/files/generated-tests.t line 20
(  DIAG  )  job  1    +------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    | PATH | GOT                          | OP | CHECK                        |
(  DIAG  )  job  1    +------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    | [8]  | use Clock ();                | eq | use Clock      ();           |
(  DIAG  )  job  1    |      |                              |    |                              |
(  DIAG  )  job  1    | [23] |         call time    => $cas | eq |         call time => $case-> |
(  DIAG  )  job  1    |      | e->{expected};               |    | {expected};                  |
(  DIAG  )  job  1    |      |                              |    |                              |
(  DIAG  )  job  1    | [52] |           call time    => $c | eq |           call time => $case |
(  DIAG  )  job  1    |      | ase->{expected};             |    | ->{expected};                |
(  DIAG  )  job  1    +------+------------------------------+----+------------------------------+
[  FAIL  ]  job  1  + grade-school: test suite matches generated
[  DEBUG ]  job  1    t/files/generated-tests.t line 20
(  DIAG  )  job  1    +------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    | PATH | GOT                          | OP | CHECK                        |
(  DIAG  )  job  1    +------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    | [23] |     "description": "Adding a | eq |     "description": "Roster i |
(  DIAG  )  job  1    |      |  student adds them to the so |    | s empty when no student is a |
(  DIAG  )  job  1    |      | rted roster",                |    | dded",                       |
(  DIAG  )  job  1    |      |                              |    |                              |
(  DIAG  )  job  1    | [24] |     "expected": [            | eq |     "expected": [],          |
(  DIAG  )  job  1    | [25] |       "Aimee"                | eq |     "input": {               |
(  DIAG  )  job  1    | [26] |     ],                       | eq |       "students": []         |
(  DIAG  )  job  1    | [27] |     "input": {               | eq |     },                       |
(  DIAG  )  job  1    | [28] |       "students": [          | eq |     "property": "roster"     |
(  DIAG  )  job  1    | [29] |         [                    | eq |   },                         |
(  DIAG  )  job  1    | [30] |           "Aimee",           | eq |   {                          |
(  DIAG  )  job  1    |      |                              |    |                              |
(  DIAG  )  job  1    | [31] |           2                  | eq |     "description": "Student  |
(  DIAG  )  job  1    |      |                              |    | is added to the roster",     |
(  DIAG  )  job  1    |      |                              |    |                              |
(  DIAG  )  job  1    | [32] |         ]                    | eq |     "expected": [            |
(  DIAG  )  job  1    | [33] |       ]                      | eq |       "Aimee"                |
(  DIAG  )  job  1    | [34] |     },                       | eq |     ],                       |
(  DIAG  )  job  1    | [35] |     "property": "roster"     | eq |     "input": {               |
(  DIAG  )  job  1    | [36] |   },                         | eq |       "students": [          |
(  DIAG  )  job  1    | [37] |   {                          | eq |         [                    |
(  DIAG  )  job  1    |      |                              |    |                              |
(  DIAG  )  job  1    | [38] |     "description": "Adding m | eq |           "Aimee",           |
(  DIAG  )  job  1    |      | ore students adds them to th |    |                              |
(  DIAG  )  job  1    |      | e sorted roster",            |    |                              |
(  DIAG  )  job  1    |      |                              |    |                              |
(  DIAG  )  job  1    | [39] |     "expected": [            | eq |           2                  |
(  DIAG  )  job  1    | [40] |       "Blair",               | eq |         ]                    |
(  DIAG  )  job  1    | [41] |       "James",               | eq |       ]                      |
(  DIAG  )  job  1    | [42] |       "Paul"                 | eq |     },                       |
(  DIAG  )  job  1    | [43] |     ],                       | eq |     "property": "roster"     |
(  DIAG  )  job  1    | [44] |     "input": {               | eq |   },                         |
(  DIAG  )  job  1    | [45] |       "students": [          | eq |   {                          |
(  DIAG  )  job  1    |      |                              |    |                              |
(  DIAG  )  job  1    | [46] |         [                    | eq |     "description": "Multiple |
(  DIAG  )  job  1    |      |                              |    |  students in the same grade  |
(  DIAG  )  job  1    |      |                              |    | are added to the roster",    |
(  DIAG  )  job  1    |      |                              |    |                              |
(  DIAG  )  job  1    | [47] |           "Blair",           | eq |     "expected": [            |
(  DIAG  )  job  1    +------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    ************************************************************
(  DIAG  )  job  1    * Stopped after 25 differences.                            *
(  DIAG  )  job  1    * Set the TS_MAX_DELTA environment var to raise the limit. *
(  DIAG  )  job  1    * Set it to 0 for no limit.                                *
(  DIAG  )  job  1    ************************************************************
[  FAIL  ]  job  1  + meetup: test suite matches generated
[  DEBUG ]  job  1    t/files/generated-tests.t line 20
(  DIAG  )  job  1    +-------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    | PATH  | GOT                          | OP | CHECK                        |
(  DIAG  )  job  1    +-------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    | [23]  |     "description": "monteent | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | h of May 2013",              |    | nth Monday is the 13th, the  |
(  DIAG  )  job  1    |       |                              |    | first day of the teenth week |
(  DIAG  )  job  1    |       |                              |    | ",                           |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [34]  |     "description": "monteent | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | h of August 2013",           |    | nth Monday is the 19th, the  |
(  DIAG  )  job  1    |       |                              |    | last day of the teenth week" |
(  DIAG  )  job  1    |       |                              |    | ,                            |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [45]  |     "description": "monteent | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | h of September 2013",        |    | nth Monday is some day in th |
(  DIAG  )  job  1    |       |                              |    | e middle of the teenth week" |
(  DIAG  )  job  1    |       |                              |    | ,                            |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [56]  |     "description": "tuesteen | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | th of March 2013",           |    | nth Tuesday is the 19th, the |
(  DIAG  )  job  1    |       |                              |    |  last day of the teenth week |
(  DIAG  )  job  1    |       |                              |    | ",                           |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [67]  |     "description": "tuesteen | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | th of April 2013",           |    | nth Tuesday is some day in t |
(  DIAG  )  job  1    |       |                              |    | he middle of the teenth week |
(  DIAG  )  job  1    |       |                              |    | ",                           |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [78]  |     "description": "tuesteen | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | th of August 2013",          |    | nth Tuesday is the 13th, the |
(  DIAG  )  job  1    |       |                              |    |  first day of the teenth wee |
(  DIAG  )  job  1    |       |                              |    | k",                          |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [89]  |     "description": "wedneste | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | enth of January 2013",       |    | nth Wednesday is some day in |
(  DIAG  )  job  1    |       |                              |    |  the middle of the teenth we |
(  DIAG  )  job  1    |       |                              |    | ek",                         |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [100] |     "description": "wedneste | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | enth of February 2013",      |    | nth Wednesday is the 13th, t |
(  DIAG  )  job  1    |       |                              |    | he first day of the teenth w |
(  DIAG  )  job  1    |       |                              |    | eek",                        |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [111] |     "description": "wedneste | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | enth of June 2013",          |    | nth Wednesday is the 19th, t |
(  DIAG  )  job  1    |       |                              |    | he last day of the teenth we |
(  DIAG  )  job  1    |       |                              |    | ek",                         |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [122] |     "description": "thurstee | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | nth of May 2013",            |    | nth Thursday is some day in  |
(  DIAG  )  job  1    |       |                              |    | the middle of the teenth wee |
(  DIAG  )  job  1    |       |                              |    | k",                          |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [133] |     "description": "thurstee | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | nth of June 2013",           |    | nth Thursday is the 13th, th |
(  DIAG  )  job  1    |       |                              |    | e first day of the teenth we |
(  DIAG  )  job  1    |       |                              |    | ek",                         |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [144] |     "description": "thurstee | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | nth of September 2013",      |    | nth Thursday is the 19th, th |
(  DIAG  )  job  1    |       |                              |    | e last day of the teenth wee |
(  DIAG  )  job  1    |       |                              |    | k",                          |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [155] |     "description": "friteent | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | h of April 2013",            |    | nth Friday is the 19th, the  |
(  DIAG  )  job  1    |       |                              |    | last day of the teenth week" |
(  DIAG  )  job  1    |       |                              |    | ,                            |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [166] |     "description": "friteent | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | h of August 2013",           |    | nth Friday is some day in th |
(  DIAG  )  job  1    |       |                              |    | e middle of the teenth week" |
(  DIAG  )  job  1    |       |                              |    | ,                            |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [177] |     "description": "friteent | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | h of September 2013",        |    | nth Friday is the 13th, the  |
(  DIAG  )  job  1    |       |                              |    | first day of the teenth week |
(  DIAG  )  job  1    |       |                              |    | ",                           |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [188] |     "description": "saturtee | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | nth of February 2013",       |    | nth Saturday is some day in  |
(  DIAG  )  job  1    |       |                              |    | the middle of the teenth wee |
(  DIAG  )  job  1    |       |                              |    | k",                          |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [199] |     "description": "saturtee | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | nth of April 2013",          |    | nth Saturday is the 13th, th |
(  DIAG  )  job  1    |       |                              |    | e first day of the teenth we |
(  DIAG  )  job  1    |       |                              |    | ek",                         |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [210] |     "description": "saturtee | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | nth of October 2013",        |    | nth Saturday is the 19th, th |
(  DIAG  )  job  1    |       |                              |    | e last day of the teenth wee |
(  DIAG  )  job  1    |       |                              |    | k",                          |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [221] |     "description": "sunteent | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | h of May 2013",              |    | nth Sunday is the 19th, the  |
(  DIAG  )  job  1    |       |                              |    | last day of the teenth week" |
(  DIAG  )  job  1    |       |                              |    | ,                            |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [232] |     "description": "sunteent | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | h of June 2013",             |    | nth Sunday is some day in th |
(  DIAG  )  job  1    |       |                              |    | e middle of the teenth week" |
(  DIAG  )  job  1    |       |                              |    | ,                            |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [243] |     "description": "sunteent | eq |     "description": "when tee |
(  DIAG  )  job  1    |       | h of October 2013",          |    | nth Sunday is the 13th, the  |
(  DIAG  )  job  1    |       |                              |    | first day of the teenth week |
(  DIAG  )  job  1    |       |                              |    | ",                           |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [254] |     "description": "first Mo | eq |     "description": "when fir |
(  DIAG  )  job  1    |       | nday of March 2013",         |    | st Monday is some day in the |
(  DIAG  )  job  1    |       |                              |    |  middle of the first week",  |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [265] |     "description": "first Mo | eq |     "description": "when fir |
(  DIAG  )  job  1    |       | nday of April 2013",         |    | st Monday is the 1st, the fi |
(  DIAG  )  job  1    |       |                              |    | rst day of the first week",  |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [276] |     "description": "first Tu | eq |     "description": "when fir |
(  DIAG  )  job  1    |       | esday of May 2013",          |    | st Tuesday is the 7th, the l |
(  DIAG  )  job  1    |       |                              |    | ast day of the first week",  |
(  DIAG  )  job  1    |       |                              |    |                              |
(  DIAG  )  job  1    | [287] |     "description": "first Tu | eq |     "description": "when fir |
(  DIAG  )  job  1    |       | esday of June 2013",         |    | st Tuesday is some day in th |
(  DIAG  )  job  1    |       |                              |    | e middle of the first week", |
(  DIAG  )  job  1    +-------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    ************************************************************
(  DIAG  )  job  1    * Stopped after 25 differences.                            *
(  DIAG  )  job  1    * Set the TS_MAX_DELTA environment var to raise the limit. *
(  DIAG  )  job  1    * Set it to 0 for no limit.                                *
(  DIAG  )  job  1    ************************************************************
[  FAIL  ]  job  1  + triangle: test suite matches generated
[  DEBUG ]  job  1    t/files/generated-tests.t line 20
(  DIAG  )  job  1    +-------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    | PATH  | GOT                          | OP | CHECK                        |
(  DIAG  )  job  1    +-------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    | [230] |     "description": "scalene  | eq |     "description": "scalene  |
(  DIAG  )  job  1    |       | triangle: two sides are equa |    | triangle: first and second s |
(  DIAG  )  job  1    |       | l",                          |    | ides are equal",             |
(  DIAG  )  job  1    +-------+------------------------------+----+------------------------------+
(  DIAG  )  job  1    Seeded srand with seed '20220912' from local date.
( FAILED )  job  1    t/files/generated-tests.t
< REASON >  job  1    Test script returned error (Err: 5)
< REASON >  job  1    Assertion failures were encountered (Count: 5)

The following jobs failed:
+--------------------------------------+---------------------------+
| Job ID                               | Test File                 |
+--------------------------------------+---------------------------+
| 01517850-329B-11ED-8796-8B0A3139F9D0 | t/files/generated-tests.t |
+--------------------------------------+---------------------------+

                                Yath Result Summary
-----------------------------------------------------------------------------------
     Fail Count: 1
     File Count: 1
Assertion Count: 50
      Wall Time: 7.96 seconds
       CPU Time: 8.05 seconds (usr: 0.45s | sys: 0.05s | cusr: 6.24s | csys: 1.31s)
      CPU Usage: 101%
    -->  Result: FAILED  <--
[0m
```

Looks like all these errors need to be fixed to get the perl5 track up and running again.

**Comment by glennj** [comment]
OK, I see what's required:
```sh
$ carton exec -- ./bin/exercise-gen.pl --all
```
and commit the changes.

**Comment by thiago-glauco** [comment]
> ```shell
> carton exec -- ./bin/exercise-gen.pl --all
> ```

@glennj What distro are you guys using? WSL Ubuntu does not install all packets.

```
Installing modules using /home/thiago/exercism-perl5/cpanfile
Successfully installed Path-Tiny-0.124
Successfully installed Try-Tiny-0.31
Successfully installed Module-Implementation-0.09
! Installing List::Util failed. See /home/thiago/.cpanm/work/1663358206.6995/build.log for details. Retry with --force to force install it.
Successfully installed List-SomeUtils-0.58
! Installing the dependencies failed: Installed version (1.50) of List::Util is not in range '1.56'
! Bailing out the installation for List-AllUtils-0.19.
! Installing Escape::Houdini failed. See /home/thiago/.cpanm/work/1663358206.6995/build.log for details. Retry with --force to force install it.
! Installing YAML::XS failed. See /home/thiago/.cpanm/work/1663358206.6995/build.log for details. Retry with --force to force install it.
Successfully installed Sub-Quote-2.006006
Successfully installed Class-Method-Modifiers-2.13
Successfully installed Moo-2.005004
Successfully installed Exporter-Tiny-1.004001
Successfully installed Type-Tiny-1.016010
Successfully installed MooseX-MungeHas-0.011
! Installing the dependencies failed: Module 'Escape::Houdini' is not installed, Module 'List::AllUtils' is not installed, Module 'YAML::XS' is not installed
! Bailing out the installation for Template-Mustache-1.4.0.
Successfully installed common-sense-3.75
Successfully installed Types-Serialiser-1.01
Successfully installed TOML-Parser-0.91
Successfully installed Importer-0.026
Successfully installed Term-Table-0.016
Successfully installed Sub-Info-0.002
Successfully installed Module-Pluggable-5.2
Successfully installed Scope-Guard-0.21
Successfully installed Test-Simple-1.302191 (upgraded from 1.302162)
Successfully installed Test2-Suite-0.000145
Successfully installed Regexp-Stringify-0.06
Successfully installed Data-Dmp-0.242
Successfully installed Modern-Perl-1.20220515
Successfully installed lib-gitroot-0.004
Successfully installed YAML-1.30
Successfully installed Sub-Exporter-Progressive-0.001013
Successfully installed B-Hooks-EndOfScope-0.26
! Installing Sub::Identify failed. See /home/thiago/.cpanm/work/1663358206.6995/build.log for details. Retry with --force to force install it.
Successfully installed Dist-CheckConflicts-0.11
Successfully installed Package-Stash-0.40
Successfully installed namespace-clean-0.27
Successfully installed namespace-autoclean-0.29
Successfully installed Perl-Tidy-20220613
! Installing the dependencies failed: Module 'Template::Mustache' is not installed
! Bailing out the installation for /home/thiago/exercism-perl5/.
32 distributions installed
Installing modules failed
```

**Comment by glennj** [comment]
You'll need to review the build.log files listed in the output to see why they failed.

I admit my perl environment is not pristine: I've installed loads of modules into my ~/perl5 over the years.

**Comment by m-dango** [comment]
Generator failure addressed in https://github.com/exercism/perl5/pull/472

**Comment by m-dango** [comment]
I think the main point I'd like to raise with this is alternating between the use of "Perl" and "Perl 5". Although the track is named "Perl 5", my personal preference leans towards "Perl". The 5 was needed due to Raku being previously known as Perl 6, but I don't think its a necessary distinction to be made now.

**Pull request closed without merging**